index.js

```diff
@@ -604,6 +701,13 @@ module.exports = function(THREE) {
             scope.dispatchEvent( endEvent );
             state = STATE.NONE;
 
+            if ( isDoubleClick( event ) === true ) {
+
+                var desiredZoomScale = +constraint.zoomScale.toFixed( 2 ) > 0.15 ? 0 : 1;
```

Comment:
Why do we still have it as 0.15? Isn't that an outdated workaround for the previous bug of zoomOut not taking zoomScale fully to 0?